### ISSUE TITLE
Show values, not types, for pixels

### ISSUE THREAD

**Issue by timholy** [timholy/ImageView.jl#53]
See https://github.com/timholy/Images.jl/issues/224#issuecomment-64386974

**Comment by rsrock** [comment]
Hrm, this issue just came up again. I was able to trace it down to the way I was slicing.

Starting with a `Gray Image` with data `512x512x192 Array{Gray{UfixedBase{Uint16,16}},3}`, if I slice with `sliceim(img, "t", 2)`, I see this issue.  The resulting datatype is:

`512x512 SubArray{Gray{UfixedBase{Uint16,16}},2,Array{Gray{UfixedBase{Uint16,16}},3},(UnitRange{Int64},UnitRange{Int64},Int64)}`

(btw, that's a mouthful!)

The weird thing is that it almost looks like I'm getting 192 pixel values for every pixel location.  I saw this while println debugging in the REPL.  Notice that the pixel location is only at the top of the list.  I didn't count, but I'd guess that the next time I see a pixel location is 192 rows later.  Shouldn't sliceim be dropping the singleton dimension?

If I slice using `img["t", 2]`, all is well when I view the result.
